share/docker/Dockerfile

```diff
@@ -26,13 +26,13 @@ ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=isolemnlysweariamuptonogood \
 
 # Install dependencies from vanilla system packages
 RUN apt-get -qqy update \
+ && apt-get -qqy remove ruby \
```

Comment:
Ruby is not installed on openjdk:11-stretch out of the box.

Reply:
You're correct. Must have been pulled in by one of the other packages I removed.